bluesky_widgets/_matplotlib_axes.py

```diff
@@ -129,12 +189,17 @@ def _on_image_added(self, event):
         image_spec = event.item
         self._add_image(image_spec)
 
+    def axes_plot(self, x, y, label, *args, **kwargs):
```

Comment:
make this private for now?  We want to over-ride it internally but maybe not commit to making it public API yet.

Reply:
and if you are going to take in `*args` make sure you use them

Reply:
What do you mean by make it private?

Reply:
Prepend `_` to the method name to signal that downstream code shouldn't call it (directly). It's still _possible_ for downstream code to use it, of course, but it signals that they _shouldn't_ and that if they do, we reserve the right to make changes that could break their code.

Reply:
Ah, I understand.